docs/package.json

```diff
@@ -13,13 +13,14 @@
     "clsx": "^1.2.1",
     "framer-motion": "^8.0.0",
     "next": "^13.1.1",
-    "nextra": "2.0.3",
-    "nextra-theme-docs": "2.0.3",
+    "nextra": "workspace:*",
+    "nextra-theme-docs": "workspace:*",
```

Comment:
this is a good improvement to add docs in the workspace ❤️ wanted to propose it too